cmd/pillard/main.go

```diff
@@ -0,0 +1,40 @@
+// Pillar is a web service for delivering community experiences
+package main
+
+import (
+	"runtime"
+
+	"github.com/ardanlabs/kit/log"
+	"github.com/ardanlabs/kit/web/app"
+
+	"github.com/coralproject/xenia/cmd/pillard/handlers"
+	"github.com/coralproject/xenia/cmd/pillard/routes"
+)
+
+// These are set by the makefile with:
+// go build -ldflags "-X main.GitVersion=8e830ff -X main.GitRevision=123123123123 -X main.BuildDate=2016-01-25"
+var (
+	GitRevision = "<unknown>"
+	GitVersion  = "<unknown>"
+	BuildDate   = "<unknown>"
+	IntVersion  = "201605091500"
+)
+
+func main() {
+	log.User("startup", "Init", "Revision     : %q", GitRevision)
+	log.User("startup", "Init", "Version      : %q", GitVersion)
+	log.User("startup", "Init", "Build Date   : %q", BuildDate)
+	log.User("startup", "Init", "Int Version  : %q", IntVersion)
+	log.User("startup", "Init", "Go Version   : %q", runtime.Version())
+	log.User("startup", "Init", "Go Compiler  : %q", runtime.Compiler)
+	log.User("startup", "Init", "Go ARCH      : %q", runtime.GOARCH)
+	log.User("startup", "Init", "Go OS        : %q", runtime.GOOS)
+
+	handlers.Version.GitRevision = GitRevision
+	handlers.Version.GitVersion = GitVersion
+	handlers.Version.BuildDate = BuildDate
+	handlers.Version.IntVersion = IntVersion
+
+	app.Run(":8080", routes.API())
```

Comment:
Should we make the port configurable, perhaps with a default that can be overwritten by env vars?  On one hand doing so would allow for more flexible deploys/environments.  On the other hand, having a standard could help simplify and prevent confusing errors.